frontend/src/queries/nodes/DataNode/dataNodeLogic.queryCancellation.test.ts

```diff
@@ -30,6 +30,13 @@ describe('dataNodeLogic - query cancellation', () => {
             },
             post: {
                 '/api/projects/997/insights/cancel/': [201],
+                '/api/projects/997/query/': async () => {
+                    return new Promise((resolve) =>
```

Comment:
IIRC we found [msw.delay](https://mswjs.io/docs/api/context/delay) was safer than `setTimeout` for cancellation tests

But some tests > no tests so 🤷